dependencies/typedb/repositories.bzl

```diff
@@ -7,8 +7,8 @@ load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 def typedb_bazel_distribution():
     git_repository(
         name = "typedb_bazel_distribution",
-        remote = "https://github.com/typedb/bazel-distribution",
-        commit = "056a8d7ede9b552d23dcfdc2d47b9395510652f4",
+        remote = "https://github.com/krishnangovindraj/bazel-distribution",
+        commit = "61e321e5a9a6dc6bffbb64233c126909a6ff89e4",
```

Comment:
Makes the Cloudsmith uploader hermetic: https://github.com/typedb/bazel-distribution/pull/421 

We might be able to remove all the python installation steps now.